backend/src/plugins/Utility/commands/CleanCmd.ts

```diff
@@ -8,6 +8,7 @@ import { GuildPluginData } from "knub";
 import { SavedMessage } from "../../../data/entities/SavedMessage";
 import { LogType } from "../../../data/LogType";
 import { allowTimeout } from "../../../RegExpRunner";
+import { snowflakeToTimestamp } from "src/utils/snowflakeToTimestamp";
```

Comment:
For consistency
```suggestion
import { snowflakeToTimestamp } from "../../../utils/snowflakeToTimestamp";
```

Reply:
Good change, though that made me notice that we dont need that line at all anymore 😅

Reply:
That was my intention all along ~~(or not)~~